lib/gpx/segment.rb

```diff
@@ -69,7 +75,10 @@ def append_point(pt)
       @bounds.min_lon = pt.lon if pt.lon < @bounds.min_lon
       @bounds.max_lat = pt.lat if pt.lat > @bounds.max_lat
       @bounds.max_lon = pt.lon if pt.lon > @bounds.max_lon
-      @distance += haversine_distance(last_pt, pt) unless last_pt.nil?
+      unless last_pt.nil?
```

Comment:
You are right, I also prefer the if notation. I just keep the original one.
PR update with if instead of unless.